app/services/ScrollingButtonMenu.js

```diff
@@ -153,16 +153,16 @@ export default class ScrollingButtonMenu extends React.Component {
                     keyboardShouldPersistTaps={keyboardShouldPersistTaps}
                 >
                     {
-                        items.map((route, i) => (
+                      (items?.length > 0) &&  items.map((route, i) => (
```

Comment:
if there is a possibility that items can be null then we add "?"
items?.map(...)